public/css/concise.css

```diff
@@ -1194,7 +1194,8 @@ address {
   color: #ffb800 !important; }
 
 .bg--warning {
-  background-color: #ffb800 !important; }
+  background-color: #E3BF44 !important;
+  border: none; }
```

Comment:
Doszedłem do wniosku, że to jest podobny odcień pomarańczowego i lepiej to zostawić w concise zamiast zaśmiecania maina